src/stream/src/task/barrier_manager/managed_state.rs

```diff
@@ -561,88 +579,4 @@ mod tests {
         );
         assert!(managed_barrier_state.epoch_barrier_state_map.is_empty());
     }
-
-    #[tokio::test]
-    async fn test_managed_state_issued_after_collect() {
```

Comment:
We ensured that collect will be called after issued, so the test is no longer needed.